app/controllers/webhooks_controller.rb

```diff
@@ -30,6 +30,21 @@ def youtube_notify
     head status: 200
   end
 
+  # Handle notification updates from getstream firehose
+  def getstream_firehose
+    notifications = JSON.parse(request.body.read)
+    # Since it may contains up to 100 per request,
+    # Letting the task run in background to eliminate
+    # API timeout and rescue possible errors
+    notifications.each do |notification|
+      unless notification['new'].empty?
+        OneSignalNotificationWorker.perform_async(notification)
```

Comment:
Agreed, we should put these in a separate queue.  That said, it might be better to handle them individually and just expand our Sidekiq workers...